cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
For tiny benefit I wonder why we need such goroutine/channel code.  What do you mean by `our real estate on console is minimal`?